opentech/apply/dashboard/templates/dashboard/dashboard.html

```diff
@@ -19,21 +23,65 @@ <h1 class="gamma heading heading--no-margin heading--bold">Dashboard</h1>
 <div class="wrapper wrapper--large wrapper--inner-space-medium">
         <div class="wrapper wrapper--large wrapper--inner-space-medium">
 
+            <h4 class="heading heading--normal">
+                Waiting for your review <span class="heading heading--submission-count">{{ in_review_count }}</span>
+            </h4>
+
+            {% if my_review.data %}
+                {% render_table my_review %}
+
+                {% if display_more %}
+                    <div class="all-submissions-table__more">
+                        <a href="{% url 'apply:submissions:list' %}?reviewers={{ request.user.pk }}">Show all</a>
+                    </div>
+                {% endif %}
+
+            {% else %}
+                <div class="grid grid--seventy-thirty wrapper wrapper--large ">
+                    <div class="reviewer-dash-box">
+                        <h5 class="reviewer-dash-box__title">Nice! You're all caught up.</h5>
+                        <a class="button button--primary" href="{% url 'apply:submissions:list' %}">Find new applications to review</a>
+                    </div>
+                    {# TODO Fill in data and update styles in future ticket #}
+                    <div>{# Since you last logged in #}</div>
+                </div>
+            {% endif %}
+
             {% if closed_rounds or open_rounds %}
                 {% include "funds/includes/round-block.html" with closed_rounds=closed_rounds open_rounds=open_rounds title=rounds_title %}
             {% endif %}
 
-            <h3>Applications to review</h3>
-            {% if in_review.data %}
-                {% render_table in_review %}
-            {% else %}
-                No reviews to complete
-            {% endif %}
         </div>
+
+        <div class="wrapper wrapper--large wrapper--inner-space-medium">
+            <div class="wrapper wrapper--large wrapper--inner-space-medium">
```

Comment:
Any particular reason for the nest divs with the same classes?

Reply:
No reason, probably just an overzealous copy paste. I can adjust that!